### PR TITLE
PLAT-3379 Local Build Changes

### PR DESCRIPTION
**Pull request by malcyL**
This PR updates the instructions to explain how to run this project against a locally running persona.talis.local.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -38,7 +38,7 @@ RUN apt-get -y autoremove && apt-get clean && apt-get autoclean && \
 RUN mkdir -p /var/talis-php
 COPY . /var/talis-php
 
-RUN echo "export PERSONA_TEST_HOST='https://staging-users.talis.com'" >> /etc/profile.d/test.sh \
+RUN echo "export PERSONA_TEST_HOST='http://persona.talis.local'" >> /etc/profile.d/test.sh \
```
Comment (malcyL) [inline Dockerfile]: The instructions previously required you to have oauth client and secret for an `su` scoped client in staging.

This PR changes this to run against persona.talis.local and details how to create the client.

File: docker-compose.yml [hunk 2]
```diff
@@ -23,21 +23,31 @@ services:
     command: /bin/bash -c "service redis-server start && source /etc/profile.d/* && ant code-check"
   test:
     image: talis/talis-php
+    network_mode: host
```
Comment (malcyL) [inline docker-compose.yml]: Needed to access persona.talis.local.

Comment (mrliptontea) [reply docker-compose.yml]: Alternatively, you could add
```
  test:
    # ...
    networks:
      - default
      - infra

# ...
networks:
  infra:
    external:
      name: talis_shared
```

Comment (malcyL) [reply docker-compose.yml]: I think the `host` setting is OK here. This isn't a docker container that's ever going to come up as part of the new docker environment for primitives and apps etc. It's just a docker container to run the tests for a client library on.

**Pull request merged**